hugo/content/050-tutorials/content/howto/prometheus/_index.md

```diff
@@ -0,0 +1,350 @@
+---
+title: Using Prometheus and Grafana to monitor K8s
+description: "How to deploy and configure Prometheus and Grafana to scrape and monitor kubelet container metrics"
```

Comment:
I used for a good reason the "scrape" wording because the parameters in the prometheus YAML are named "scrape_interval" and "scrape_timeout". But anyhow - I change it.

Reply:
9c1bede8c6d18281e36a5f296151de7dae86628f